src/index.html

```diff
@@ -9,10 +9,56 @@
     <title>Stars</title>
     <link
       rel="stylesheet"
-      href="./style.css"
+      href="style.css"
     />
```

Comment:
The attributes in the <link> tag should start on a new line with a 2-space indentation relative to the tag for consistency and readability.